docs/manage/software/deploy-software.md

```diff
@@ -25,17 +25,36 @@ For microcontrollers, see [Over-the-air firmware updates](/operate/get-started/o
 ## Prerequisites
 
 Start by [setting up one machine](/operate/get-started/setup/).
-Then, [configure any hardware and software resources](/operate/get-started/supported-hardware/) that you want to use with your machine and drive with your control logic.
+Once your machine is connected to the Viam app, return to this page.
 
-## Configure your software
+## Create a fragment
+
+Viam has a built-in tool called _{{< glossary_tooltip term_id="fragment" text="fragments" >}}_ for using the same configuration on multiple machines.
```

Comment:
```suggestion
Viam has a built-in tool called _{{< glossary_tooltip term_id="fragment" text="fragments" >}}_ that enable you to reuse one configuration across multiple machines.
```
[opt] this occurred to me the third time i read this sentence, might be better? eliminating gerunds and making the phrasing more user-centric seems like a win, in objective terms. take it or leave it